jenkins-pipelines/rolling-upgrade-ubuntu22.04.jenkinsfile

```diff
@@ -0,0 +1,15 @@
+#!groovy
```

Comment:
didn't we have rolling upgrade test for ubuntu22 till now?

Reply:
no, we did not have it, hence i'm adding it in this PR